app.yaml

```diff
@@ -1,6 +1,6 @@
   
 env: standard
-service: application-monitoring-react
+service: -<SERVICE_NAME_HERE>-
```

Comment:
this change helps prevent people from accidentally overwriting the master admin instance.